examples/static-file-server/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+use axum::http::StatusCode as SC;
+use axum::{body::Body, http::Request};
+use http_body_util::BodyExt;
+use tower::ServiceExt;
+
+const INDEX_HTML: &str = include_str!("../assets/index.html");
+const SCRIPT_JS: &str = include_str!("../assets/script.js");
+
+const JS: &str = "text/javascript";
+const HTML: &str = "text/html";
```

Comment:
Personally, I'd prefer more descriptive names like `INDEX_HTML_CONTENT` and `CONTENT_TYPE_JAVASCRIPT`. It's much clearer what you're doing especially in calls like `check(app(), "/assets/", SC::OK, HTML, INDEX_HTML).await;`.

Also use `ContentType` as the type instead of `&str` for the latter two. I don't think it can be `const` then but `static` really is good enough.

Reply:
Right so I shortened all the names to let `fmt` put them (most of them) on one line, but I guess it is better to have longer names even if that means the test are then in multiple lines.

Or maybe you have an idea how to arrange things to have both?

Reply:
I don't see how to have both, but more lines with more descriptive names are to me better than the alternative.